### PR TITLE
added additional connectors for presto-spark-package

### PR DESCRIPTION
**Pull request by Ravion**
Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== RELEASE NOTES ==

General Changes
* ...
* ...

Hive Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-spark-package/pom.xml [hunk 1]
```diff
@@ -100,6 +100,50 @@
             <type>zip</type>
             <scope>provided</scope>
         </dependency>
+         <dependency>
```
Comment (arhimondr) [inline presto-spark-package/pom.xml:103]: nit: pleas reformat the file to make the spaces align

File: presto-spark-package/pom.xml [hunk 2]
```diff
@@ -100,6 +100,50 @@
...
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-sqlserver</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <scope>provided</scope>
+        </dependency>
+
```
Comment (arhimondr) [inline presto-spark-package/pom.xml:145]: nit: unnecessary empty lines

File: presto-spark-package/src/main/assembly/presto.xml [hunk 3]
```diff
@@ -60,5 +60,29 @@
             <directory>${project.build.directory}/dependency/presto-postgresql-${project.version}</directory>
             <outputDirectory>plugin/postgresql</outputDirectory>
         </fileSet>
+         <fileSet>
```
Comment (arhimondr) [inline presto-spark-package/src/main/assembly/presto.xml:63]: ditto

**Comment by arhimondr** [comment]
@Ravion Thanks for the update. Could you please now squash the commit together and change the commit message to `Add additional connectors for presto-spark-package`. Once done I can merge it.

**Comment by Ravion** [comment]
Done, kindly review 
git reset --soft HEAD~3
git commit -m "Add additional connectors for presto-spark-package"
git push -f

**Comment by arhimondr** [comment]
@Ravion Thanks for the contribution

**Comment by Ravion** [comment]
My pleasure. Can I open another pull request for writing documentation on
this?
My proposal is to add a section 2.6 ( under section 2 installation of
exiting doc) , "Running Presto on Spark". Just like server jars are
available for deploying Presto Server, we need to make spark package
artifacts also downloadable. Please advise.

By the by, thanks to you and team for inspiration. I have published this:

https://medium.com/@ravishankar.nair/the-ultimate-duo-in-distributed-computing-prestodb-running-on-spark-b63d0e567eeb


Best, Ravion

**Comment by arhimondr** [comment]
> My pleasure. Can I open another pull request for writing documentation on
this?

Please do. We really short on resources right now, so we don't know when we are going to get 
 to that. If you can write the documentation that would be an enormous help for us.

> My proposal is to add a section 2.6 ( under section 2 installation of
exiting doc) , "Running Presto on Spark". Just like server jars are
available for deploying Presto Server, we need to make spark package
artifacts also downloadable. Please advise.

Yeah, that makes sense. Please add a new section. The launcher jar and tar.gz package are available in maven central, same as the presto-server package. Here is an example of how to link it: https://github.com/prestodb/presto/blame/master/presto-docs/src/main/sphinx/installation/deployment.rst#L8 (here is the script with the maven_download function definition: https://github.com/prestodb/presto/blob/6dc08b619cf4a81f5f3ebaceb23e134653a82f3b/presto-docs/src/main/sphinx/ext/download.py#L35)

You can also consider editing our web site to include download urls: https://github.com/prestodb/prestodb.github.io/blob/source/website/static/download.html#L51

Thanks for your interested in the project and willing to contribute. I will be more then happy to do the review and answer any additional questions.

**Comment by Ravion** [comment]
@arhimondr Submitted dded documentation for Presto on Spark
#14735
Kindly review

**Pull request merged**